mech_client/interact.py

```diff
@@ -56,117 +64,152 @@
     "WEBSOCKET_ENDPOINT",
     "wss://rpc.eu-central-2.gateway.fm/ws/v4/gnosis/non-archival/mainnet",
 )
+GNOSISSCAN_API_URL = "https://api.gnosisscan.io/api?module=contract&action=getabi&address={contract_address}"
 
 # Ignore a specific warning message
 warnings.filterwarnings("ignore", "The log with transaction hash.*")
 
 
-def check_for_tools(tool: str) -> Optional[int]:
+def get_contract(contract_address: str, ledger_api: EthereumApi) -> Web3Contract:
+    """Returns a contract instance."""
+    abi_request_url = GNOSISSCAN_API_URL.format(contract_address=contract_address)
+    response = requests.get(abi_request_url).json()
+    abi = json.loads(response["result"])
+    return ledger_api.get_contract_instance(
+        {"abi": abi, "bytecode": "0x"}, contract_address
+    )
+
+
+def _tool_selector_prompt(available_tools: List[str]) -> str:
+    """Tool selector prompt."""
+
+    tool_col_len = len(max(available_tools))
```

Comment:
```python
len(max(["c", "ab"])) == len("c") == 1
```

Would that be the desired column length or would it be `len("ab") == 2`?

Reply:
Fixed [43dd037](https://github.com/valory-xyz/mech-client/pull/4/commits/43dd0377478a109a0a0c27d30a27174a75ff8e1c)